torch/_dynamo/variables/functions.py

```diff
@@ -122,6 +122,13 @@ def __init__(self, fn, is_constant=False, **kwargs):
         ), f"expected FunctionType found {typestr(fn)} {fn}"
         # unpack @torch._dynamo.optimize()(fn) wrapped function
         fn = inspect.getattr_static(fn, "_torchdynamo_inline", fn)
+        if istype(fn, types.MethodType):
+            # when we record the original fn, it may be a bound method such as .forward
+            # however, NNModuleVariable.call_function special-cases .forward to be called
```

Comment:
i'm working on pushing out another PR on the top of the stack to reverse this and try using Method in nnmodule, naive approach bombed out somewhere..